day-3.lisp

```diff
@@ -0,0 +1,33 @@
+(defpackage aoc-day-3
+  (:use :cl)
+  (:export :answers))
+(in-package :aoc-day-3)
+
+(defun cons+ (a b)
+  (cons (+ (car a) (car b)) (+ (cdr a) (cdr b))))
+
+(defun answers (input-file)
+  (let ((input (uiop:read-file-string input-file))
+        (pos '(0 . 0)) (pos-santa '(0 . 0)) (pos-robot '(0 . 0))
+        (visited '((0 . 0)))
+        (santa-visited '((0 . 0)))
+        (robot-visited '((0 . 0)))
+        (santa-turn t))
+    (map nil (lambda (c)
```

Comment:
Why is this indented more than the ``let`` on line 10?. If I'm counting brackets correctly this map is second statement and not a parameter for the ``let``

Reply:
The `map` is a parameter for the `let`. The first parameter of the `let` is always the variable declarations/definitions, and the rest of the parameters are any number of forms to execute, returning the result of the last form. In this case, there are two forms - the `map` and the return of value of `input`.